ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.css

```diff
@@ -0,0 +1,8 @@
+.vh-border {
```

Comment:
are these specific to the `access to a room` or can they be moved up to the general css?

Reply:
Yes, it's specific